src/main/java/org/fcrepo/audit/InternalAuditor.java

```diff
@@ -231,7 +226,18 @@ public void createAuditNode(final FedoraEvent event) throws IOException {
     }
 
     @VisibleForTesting
+    protected String getEventPath(final String eventID) {
+        if (!eventID.startsWith("urn:uuid:")) {
+            throw new IllegalArgumentException("Event ID must be a 'urn:uuid:'" + eventID);
+        }
+        return pathMinter.get(eventID.substring("urn:uuid:".length()));
```

Comment:
The `HierarchicalIdentifierSupplier` currently extends `java.util.function.Supplier` (via `UniqueValueSupplier`). That is, it has a single method: `String get()`. It seems that here you would be extending that interface to include `String get(String)`. I wonder whether we should be extending that interface (which to me seems to diverge from the existing `Supplier` semantics) or simply have a utility function in the audit package that copies those two lines of code from `HierarchicalIdentifierSupplier`. My inclination is to do the latter as it keeps the kernel-api interface cleaner. Are there other thoughts on this?